apps/backend/db/migrations/20250915191132_team_invites.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+export const up = async (knex) => {
+  await knex.schema.createTable("team_invites", (table) => {
+    table.dateTime("createdAt").notNullable().defaultTo(knex.fn.now());
```

Comment:
Cool to have default values. I think that the only one table that have it. We should have it on all of createdAt and updatedAt IMO